src/App.tsx

```diff
@@ -62,6 +62,16 @@ const App: FC = () => {
         })
       }
     }
+    if (CLOUD && isFlagEnabled('coveo')) {
+      const script = document.createElement('script')
```

Comment:
why are we dynamically injecting code like this?

Reply:
The dynamic injection is just to get this feature out behind a flag in order to demonstrate it to the Coveo team as a POC. Once this is done, we can work on getting a more stable / futureproof implementation in our code if necessary. I'll be in contact with Coveo after this is displayed to discuss alternative implementation options if needed.

Reply:
While this does represent a diversion from our usual implementation patterns, it should be noted that this suggestion was provided directly by the Coveo team and allows us to iterate quickly on the feature within a testing context and a public key that they can immediately invalidate should anything arise that is of concern.

Reply:
so we're going to remove it / relocate it once we've decided it's worth more investment?